README.md

```diff
@@ -4,3 +4,26 @@ Installers for MNE-Python for macOS, Windows, and Linux.
 
 ⚠️ Nothing to see here for now, please move along and check back again in a few
 days. ⚠️
+
+
+
+## Installation
+
+To install MNE-Python simply double click the installer.
+You can customise some parameters, but the default values should suit most users.
+Once completed, a conda evironment should be installed in your home directory.
```

Comment:
In fact, on Windows and Linux, users should be able to choose to install into whichever directory they want.

On the Mac, with the graphical installer, the directory is pre-defined (side note: it is also possible to create a `.sh` installer for the Mac that offers the same flexibility as the Linux installer, but I decided against this for now. Willing to reconsider if you think that was not a good decision)

At any rate, currently the graphical Mac installer dumps everything to `~/mne-python`. I want to change this to `~/mne-python/0.23.0` (or which ever version the user installs) to make it possible to have multiple versions installed in parallel (see #7). Do you think this is actually a good idea?

Reply:
> but I decided against this

Agree, simpler is best. If someone wants anything more they should use the traditional python install path and get all the flexibility they want.

> make it possible to have multiple versions installed in parallel (see #7). Do you think this is actually a good idea

Yes, this would be widely used with my colleagues. People like to chose a version at the start of a project and stick with it (unless bugs of course) until the project ends. And often there is multiple projects going at once. Long story short, multiple versions is good.